packages/labs/analyzer/src/lib/javascript/jsdoc.ts

```diff
@@ -151,32 +162,45 @@ function makeDashParseDiagnostic(
  * * @cssProp [--name=default] description
  * * @cssProp [--name=default] - description
  * * @cssProp [--name=default]: description
+ * * @cssprop {<color>} [--name=default]
+ * * @cssprop {<color>} [--name=default] description
+ * * @cssprop {<color>} [--name=default] - description
+ * * @cssprop {<color>} [--name=default]: description
  */
 export const parseNamedJSDocInfo = (
   tag: ts.JSDocTag,
   analyzer: AnalyzerInterface,
   requireDash = false
-): NamedDescribed | undefined => {
+): NamedDescribed | NamedDescribedSyntax | undefined => {
   const comment = getJSDocTagComment(tag, analyzer);
   if (comment == undefined) {
     return undefined;
   }
+
   const regex = requireDash ? parseNameDashDescRE : parseNameDescRE;
   const nameDesc = comment.match(regex);
   if (nameDesc === null) {
     analyzer.addDiagnostic(
       makeDashParseDiagnostic(analyzer.typescript, tag, requireDash)
     );
+
     return undefined;
   }
-  const {name, description, defaultValue} = nameDesc.groups!;
-  const info: NamedDescribed = {name};
+  const {name, description, defaultValue, type} = nameDesc.groups!;
+
+  const info: NamedDescribed | NamedDescribedSyntax = {name};
   if (description?.length > 0) {
     info.description = normalizeLineEndings(description);
   }
   if (defaultValue?.length > 0) {
     info.default = defaultValue;
   }
+  if (
+    tag.tagName.text.toLowerCase().startsWith('cssprop') &&
```

Comment:
Rather than special-casing cssprop, can we treat the CSS syntax as a type? In the jsdoc format it comes in exactly the same place as type. Then we could reuse parseNamedTypedJSDocInfo for css properties, and just move the type into a syntax field of a CSSPropertyInfo object.